src/order/rel_iso.lean

```diff
@@ -237,6 +327,8 @@ instance : has_coe_to_fun (r ≃r s) := ⟨λ _, α → β, λ f, f⟩
 
 @[simp] lemma coe_coe_fn (f : r ≃r s) : ((f : r ↪r s) : α → β) = f := rfl
 
+@[simp] lemma coe_coe_coe_fn (f : r ≃r s) : (((f : r ↪r s) : r →r s) : α → β) = f := rfl
```

Comment:
`simp` can already do this, so this lemma might not be necessary.

Reply:
I'm surprised the linter didn't complain at this.